lint/rule_test.go

```diff
@@ -0,0 +1,396 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lint
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/jhump/protoreflect/desc"
+	"github.com/jhump/protoreflect/desc/builder"
+)
+
+func TestFileRule(t *testing.T) {
+	// Create a file descriptor with nothing in it.
+	fd, err := builder.NewFile("test.proto").Build()
+	if err != nil {
+		t.Fatalf("Could not build file descriptor.")
```

Comment:
Print the `err` as well.

Reply:
Done, but this really does not matter as there is no user input or other variance, and this will never actually error.